### PR TITLE
fix: handle boostable lp channels

### PR DESCRIPTION
**Pull request by GabrielBuragev**

### CODE REVIEW

File: packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts [hunk 1]
```diff
@@ -12,20 +13,38 @@ export const insufficientBoostLiquiditySchema = z.object({
 export const insufficientBoostLiquidity = async ({ prisma, event, block }: EventHandlerArgs) => {
   const { channelId, asset, amountAttempted } = insufficientBoostLiquiditySchema.parse(event.args);
 
-  const channel = await prisma.swapDepositChannel.findFirstOrThrow({
-    where: { channelId, srcAsset: asset },
+  const depositChannel = await prisma.depositChannel.findFirst({
+    where: { channelId, srcChain: assetConstants[asset].chain },
```
Comment (acdibble) [inline packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts:17]:
```suggestion
    where: { channelId, srcChain: assetConstants[asset].chain, isSwapping: true },
```

File: packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts [hunk 2]
```diff
@@ -12,20 +13,38 @@ export const insufficientBoostLiquiditySchema = z.object({
...
-      swapDepositChannel: {
-        connect: {
-          id: channel.id,
+  if (!depositChannel) {
+    throw new Error(
+      `InsufficientBoostLiquidity: Deposit channel not found for asset ${asset} and channelId ${channelId}`,
+    );
+  }
+
+  if (depositChannel.isSwapping) {
```
Comment (acdibble) [inline packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts:29]:
```suggestion
```

**Review by acdibble — APPROVED** [review]
slight simplification

File: packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts (hunk 1 above)
Comment (GabrielBuragev) [reply packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts:17]: I wanted to still keep the assertion and check for an existing deposit channel disregarding the type of the channel

**Pull request merged**